cfgov/unprocessed/apps/regulations3k/css/notification.less

```diff
@@ -0,0 +1,7 @@
+/* ==========================================================================
+   Regulations 3000
+   Notifications on landing pages
+   ========================================================================== */
+.o-notification {
```

Comment:
Should we be more specific with the selector here? I think this would target any `.o-notification` element on any regulations page, which might interfere with the notifications on section pages about a version not being current law. Should we add our own class to these landing page notifications (like we did with an inline interpretation class for expandables)?

Reply:
`o-notification` is another selector that comes about because I'm envisioning this possibly being moved into the v1 organisms package, and I was trying to follow convention there.

Reply:
Ah, got it. I was thinking that `.o-notification` was our standard wrapper for notifications, but those are `.m-notification` elements at the molecule level. Never mind—sorry about that!

In that case, this makes sense to me (though @contolini is much more up to date on our atomic design conventions than I am).